### PR TITLE
Accessibility and Respnosnive Layout

### LINKED ISSUE #15: Accessibility Audit
- run an accesibility audit and fix errors to get to 100%

### LINKED ISSUE #16: Responsive Design:
- add media queries to insure this is viewable on mobile
- make sure measurements are flexible enough to adapt to various screen sizes

### PR DESCRIPTION
**Pull request by nathanielmillard**
###### What does this PR do?  
- passes the lighthouse accessibility audit with 100%
- adds labels to nput fiels
- add responsive formatting on the scss

###### Where should the reviewer start?  
- npm start and opening the server url

###### How should this be manually tested?  
- running a lighthouse audit
- changing the view port size

###### Any background context you want to provide?  
- The lighthouse audit has given me an error that heading are not sequential and then doesn't between different runs with no change to the code....

###### What are the relevant tickets?  
- closes #15 
- closes #16 

###### Screenshots (if appropriate)  
Screen Shot 2020-09-21 at 6 50 11 PM

Screen Shot 2020-09-22 at 1 26 29 PM

Sep-22-2020 13-37-19

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -32,81 +34,110 @@ header {
...
   }
-}
-
-img {
-  height: 100px;
+  input {
+    font-family: $bodyfont;
+    border: none;
+    height: 3vh;
+    background-color: $lightc;
```
Comment (MichaelEWalker87) [inline src/css/base.scss:41]: I might look to change these so be more descriptive for the colors

File: src/css/base.scss [hunk 2]
```diff
@@ -32,81 +34,110 @@ header {
...
+  grid-template-rows: minmax(100px, 75%) auto;
 }
 
 form {
-  @include gridArea($highmedc, $darkw)
-  @include flexControl(column)
-  align-content: center;
-  justify-content: space-between;
+  @include gridArea($highmedc, $darkw, $lowmedw);
+  @include flexControl(column);
```
Comment (MichaelEWalker87) [inline src/css/base.scss:56]: I love how you use the includes to clean up the file

File: src/css/base.scss [hunk 3]
```diff
@@ -132,3 +163,58 @@ form {
...
+      h6 {
+        font-size: .2rem;
+      }
+    }
+  }
+  .previous-trips {
+    min-height: 20vh;
+  }
+
+}
```
Comment (MichaelEWalker87) [inline src/css/base.scss:220]: 500 px seems a bit small what phones does that work on?

File: src/domUpdates.js [hunk 4]
```diff
@@ -36,8 +36,8 @@ const domUpdates = {
     })
     return `
     <div class="trip-card ${trip.status}">
-      <h5>Where are you headed?</h5>
-      <h6 class = "country-title"> ${destination.destination} </h6>
+      <h4>Where are you headed?</h4>
```
Comment (MichaelEWalker87) [inline src/domUpdates.js:39]: nice change to bigger text

**Review by MichaelEWalker87 — APPROVED** [review]
Looks Great! good work on the SASS and getting lines buttoned up

File: src/css/base.scss (hunk 1 above)
Comment (nathanielmillard) [reply src/css/base.scss:41]: Totally, my thought process was having them be a cool and warm hue of whatever value they occupy, but thats a good thought for my refactor and comb over now that they are less deisgn place holders and more actual decisions.

File: src/css/base.scss (hunk 3 above)
Comment (nathanielmillard) [reply src/css/base.scss:220]: 500 felt like a good cut off for most phones actually, all iphones fall under 500 as well as as most androids and even the pixel XL. 
The only space that might be a little crammed is a small tablet, but even they they would have a rotation features to get a wider vw.

File: src/domUpdates.js (hunk 4 above)
Comment (nathanielmillard) [reply src/domUpdates.js:39]: Strangely enough this was just to make sure I had consecutive header elements, which I am still a little blurry on the accessibility benefits of.

**Pull request merged**